about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="@codedit334">
+    <meta name="description" content="Capstone project 1: Camping website">
+    <meta name="keywords" content="Camp, Website, Responsive">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style/style.css">
+    <title>About All In Campsters</title>
+</head>
+
+<body>
+    <main>
+        <div class="navBar">
+            <div class="logo">
+                <a href="./index.html"><img src="./assets/AIC.svg" alt="Logo"></a>
+                
+            </div>
+            <div class="space"></div>
+            <ul class="menu">
+                <li class="x">&times;</li><br>
+                <li ><a href="./index.html">Index</a></li>
```

Comment:
- [ ] [OPTIONA] It would be nice if you change name of your first page(`"Index"`) to descriptive one 👍🏼 

![image](https://user-images.githubusercontent.com/63915024/217051889-3ec92375-c433-4d8f-a560-7c6dedb7f21a.png)